spec/rubocop/cop/rspec/aggregate_examples_spec.rb

```diff
@@ -0,0 +1,790 @@
+RSpec.describe RuboCop::Cop::RSpec::AggregateExamples, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) { {} }
+
+  shared_examples 'detects and autocorrects' do |offensive_source, good_source|
+    it 'does not detect an offense in good_source code' do
+      expect_no_offenses(good_source)
+    end
+
+    it 'detects an offense in offensive source code' do
+      expect_offense(offensive_source)
+    end
+
+    bad_code_without_message =
+      offensive_source.lines.delete_if { |line| line =~ /\A\s*\^/ }.join
+    include_examples 'autocorrect', bad_code_without_message, good_source
```

Comment:
The output of this included example is excessive:
![image](https://user-images.githubusercontent.com/6916/52165698-a9ae4b00-2736-11e9-8c5a-b348797c5c0c.png)

Is that ok?
I've checked, a similar output comes from e.g. `rspec spec/rubocop/cop/rspec/predicate_matcher_spec.rb:342`.

Reply:
I think it’s fine. Anyway it’s an unrelated issue that should be fixed in a separate PR.